_html/scss/modules/_sponsors-categories.scss

```diff
@@ -328,14 +328,20 @@
             }
 
             .l-row {
-                margin: 0 (-$global-margin) * .5;
+                margin:    0 (-$global-margin) * .5;
+                font-size: 0;
             }
 
             .l-col-2,
             .l-col-4 {
                 margin-bottom: $global-margin * 3;
             }
 
+            .l-col {
```

Comment:
Please don't overwrite layout classes. Use another class for this or change the HTML structure.